src/app/views/admissions/cohort-form/index.jsx

```diff
@@ -1,447 +1,529 @@
-        .then((data) => data)
-        .catch((error) => console.log(error));
+    const makePrivate = () => {
+        bc.admissions().updateCohort(cohort.id, { ...cohort, private: !cohort.private, syllabus:`${cohort.syllabus.certificate.slug}.v${cohort.syllabus.version}`})
+                .then((data) => {
+                    setCohort({...cohort, private: !cohort.private})
+                })
+                .catch(error => console.log(error))
+        
     }
-  };
 
-  const ProfileSchema = Yup.object().shape({
-    current_day: Yup.number()
-      .max(maxSyllabusDays, `You can not set a day greater than ${maxSyllabusDays}`)
-      .required('Please enter a day'),
-  });
+    const updateCohort = (values) => {
+        const { ending_date, ...rest } = values
+        if (values.never_ends) bc.admissions().updateCohort(cohort.id, { ...rest, private: cohort.private, ending_date:null })
+            .then((data) => data)
+            .catch(error => console.log(error))
+        else {
+            bc.admissions().updateCohort(cohort.id, { ...values, private: cohort.private })
+                .then((data) => data)
+                .catch(error => console.log(error))
+        }
+    }
+    
+    const ProfileSchema = Yup.object().shape({
+        current_day: Yup.number()
+            .max(maxSyllabusDays, `You can not set a day greater than ${maxSyllabusDays}`)
+            .required("Please enter a day")
+    });
 
-  return (
-    <>
-      <div className="m-sm-30">
-        <div className="flex flex-wrap justify-between mb-6">
-          <div>
-            <h3 className="mt-0 mb-4 font-medium text-28">
-              Cohort:
-              {slug}
-            </h3>
-            <div className="flex">
-              <div
-                aria-hidden="true"
-                className="px-3 text-11 py-3px border-radius-4 text-white bg-green "
-                onClick={() => setStageDialog(true)}
-                style={{ cursor: 'pointer' }}
-              >
-                {cohort && cohort.stage}
-              </div>
+
+    return (
+        <>
+            <div className="m-sm-30">
+                <div className="flex flex-wrap justify-between mb-6">
+                    <div>
+                        <h3 className="mt-0 mb-4 font-medium text-28">Cohort: {slug}</h3>
+                        <div className="flex">
+                            <div className="px-3 text-11 py-3px border-radius-4 text-white bg-green " onClick={() => setStageDialog(true)} style={{ cursor: "pointer" }}>
+                                {cohort && cohort.stage}
+                            </div>
+                        </div>
+                    </div>
+                    {isLoading && <MatxLoading />}
+                    <DowndownMenu
+                        options={options}
+                        icon="more_horiz"
+                        onSelect={({ value }) => {
+                            value === "current_day"
```

Comment:
Eslint prioritizes good practices [`no-unused-expressions`](https://eslint.org/docs/rules/no-unused-expressions) and here many ternary functions are used that could be reduced to a simple function

`proposal:`
```javascript
const dropDownMenuOption = (option) => {
  if (option === 'current_day') setCohortDayDialog(true);
  else setCohortDayDialog(false);
  if (option === 'stage') setStageDialog(true);
  else setStageDialog(false);
  if (option === 'new_survey') setSurveyDialog(true);
  else setSurveyDialog(false);
  if (option === 'privacy') makePrivate();
};
```

if you have a better idea feel free to use it